moveit_ros/planning/planning_request_adapter_plugins/src/add_time_optimal_parameterization.cpp

```diff
@@ -47,12 +47,48 @@ using namespace trajectory_processing;
 class AddTimeOptimalParameterization : public planning_request_adapter::PlanningRequestAdapter
 {
 public:
+  
+  static const std::string RESAMPLE_PARAM_NAME;
```

Comment:
I'd change these to `constexpr char PARAM[]` and move it out of this class and to the top of the surrounding namespace.

Reply:
ping @drewhamiltonasdf